src/Coin.cpp

```diff
@@ -74,7 +74,11 @@ bool TW::validateAddress(TWCoinType coin, const std::string& string) {
         return Decred::Address::isValid(string);
 
     case TWCoinTypeDogecoin:
-        return Bitcoin::Address::isValid(string, {{TWP2PKHPrefixDogecoin}, {TWP2SHPrefixDogecoin}});
+        return Bitcoin::Address::isValid(string, {{TWP2PKHPrefixD}, {TWP2SHPrefixDogecoin}});
+
+    case TWCoinTypeDigiByte:
+        return Bitcoin::SegwitAddress::isValid(string, HRP_DIGIBYTE) ||
+               Bitcoin::Address::isValid(string, {{TWP2PKHPrefixD}, {TWP2SHPrefixBitcoin}});
```

Comment:
We need also to add p2sh prefix S here

Reply:
Done in 2f76ee734e60a9d39c75665fe54524b90e1b924c